lib/tools.js

```diff
@@ -28,13 +28,19 @@ function noop () {}
 function genLog (z) {
   return function LOG (o, ...n) {
     if (typeof o === 'object' && o !== null) {
+      const orig = o
       if (o.method && o.headers && o.socket) {
         o = mapHttpRequest(o)
       } else if (typeof o.setHeader === 'function') {
         o = mapHttpResponse(o)
       }
       this[writeSym](o, format(null, n, this[formatOptsSym]), z)
-    } else this[writeSym](null, format(o, n, this[formatOptsSym]), z)
+      return orig
```

Comment:
if we return interpolated strings you could argue we should return mapped req/res?

Reply:
Well, the idea is that you can inline your logging statements when doing assignments, etc, so I don't think you'd want to return the mapped objects.  You'd want to return the originals.  It was clear that if the first argument was a formatting string, that's probably _not_ what the caller would want returned; they'd probably want the interpolated string (or ignore the returned value).  Returning the interpolated string seemed like the most reasonable thing in this case.  The slightly odd one is when you have something like `log.info('foo', 'bar')`.  Did the caller want `'foo'` or `'foo bar'` returned?  For consistency with formatted strings, I went with returning `'foo bar'`.

Reply:
yeah I think these edge cases and unknowns is another -1 from me :( sorry I don't think it's a good idea

Reply:
@davidmarkclements I'm ok with just returning the first argument, period.  That would simplify things.  WDYT?